### ISSUE TITLE
Implement Heatmaps

### ISSUE THREAD

**Issue by gamalielpalomo** [CityScienceLab-GDL/cityscope-ccu#8]
Add heatmaps to show the changes in some indicators

**Comment by agrignard** [comment]
Concerning heatmaps and depending on what you want to do and where you want to display it

1. For a display on the table from GAMA You can have a look in the GAMA template library here ```models/Visualization and User Interaction/Visualization/Building Heatmap.gaml``` Gama now has a new feature that allows you to define a layer ```mesh```on top of your agent to show any heatmap. 

2. For a display in the CityScope FrontEnd you can also easily send information from GAMA to show heatmap following this link https://cityscope.media.mit.edu/modules/GAMA/#basic-heatmap-indicator

**Comment by gamalielpalomo** [comment]
@agrignard Is there a documentation page about how to use field and mesh?
We want to use this kind of visualization for the indicators
![image](https://user-images.githubusercontent.com/19692636/160451412-98410294-b31d-4f3e-827a-63299b061e7f.png)

**Comment by agrignard** [comment]
@gamalielpalomo everything related to mesh and heatmap is implemented (and somehow self documented) here
models/Visualization and User Interaction/Visualization/Building Heatmap.gaml

Is this enough for CityScope Guadalajara ? What else do you need? Don't hesitate to point me to a specific part of your code if needed

**Comment by gamalielpalomo** [comment]
I just have pushed a work that I have done related to the heatmap:
I use the grid species but I dont like at all how it looks. 
![image](https://user-images.githubusercontent.com/19692636/160796448-071141e8-b170-47d8-84db-889098b13cfa.png)

**Comment by agrignard** [comment]
Yes that's why we have now ```mesh``` and ```heatmap``` as an alternative of using grid

**Comment by gamalielpalomo** [comment]
What is the name of the model which implements mesh?

**Comment by agrignard** [comment]
As already mention in my first anwser it's implemented here
```models/Visualization and User Interaction/Visualization/Building Heatmap.gaml ```

**Comment by gamalielpalomo** [comment]
Currently doing some work in this aspect to enhance our heatmaps
